TECHNICAL_VALUES_AND_PRIORITIZATION.md

```diff
@@ -0,0 +1,65 @@
+# Technical Values and Priorities
+
+This document outlines the project's technical values and relative
+prioritization used when making tradeoffs.
+
+## Values and priority level
+
+This is only a subset of the overall priorities, listing only
+those which are at the top of the priority list. Anything on this
+list is very important for the project despite the relative priorities shown.
+
+* Priority 1 - Developer experience
+* Priority 2 - Stability
+* Priority 3 - Operational qualities
+* Priority 4 - the Node.js maintainer experience
+* Priority 5 - Technology and API currency
```

Comment:
Is the idea here that Priority 1 is the most important and Priority 5 is less important than the preceding 4 priorities? Whatever the answer to that is, we could probably stand to make that super clear. (Is that what the paragraph I propose deleting was supposed to communicate?)

Reply:
Priority 1 is more important than Priority 2 and so on. (so yes to your first question)  Not to say that it makes a decision always black and white, as we may concede some impact to priority 1 if the positive impact is worth it in one of the lower priorities ones. But if there is equal value/loss we'd take the value in the one with the higher priority.